partials/language-specific-deploy/php.md

```diff
@@ -210,14 +210,16 @@ $dbh = new PDO(
 
 ## Composer
 
-We support Composer build out of the box. You just need to provide a `composer.json` file in the root of your repository and we will run `composer.phar install` for you.
+We support Composer build out of the box. You just need to provide a `composer.json` file in the root of your repository and we will run `composer.phar install --no-ansi --no-progress --no-interaction --no-dev` for you.
 
 You can also set the `CC_COMPOSER_VERSION` to `1` or `2` to select the composer version to use.
 
 {{< alert "info" "Note:" >}}
     <p>If you encounter any issues, add your own `composer.phar` file in the root of your repository which will override the version we use.</p>
 {{< /alert >}}
 
+You can perform your own `composer.phar install` by using the [Post Build hook]({{< ref "develop/build-hooks.md#post-build-cc_post_build_hook" >}}) if for example you need to install dev dependencies.
```

Comment:
Can you also add a comment like `if you need to install development dependencies automatically, please let us know by opening a support ticket. This feature may be implemented using an environment variable in the future if this is often asked`.

That's what we did for nodejs at some point and we ended up adding an environment variable to let users control this. In the PHP world in may be something less needed but if it is, we can do it too.

Reply:
Well it is an often asked feature, wouldn't it be better to create an issue ?

Reply:
Oh, okay. The issue already exists on our side but if we have users asking for it, then it can be a good metric to prioritize it.

Reply:
Yep